assets/src/blocks/Articles/ArticlesEditor.js

```diff
@@ -19,88 +18,86 @@ const {__} = wp.i18n;
+          value={attributes.read_more_link}
+          onChange={toAttribute('read_more_link')}
+        />
+        <CheckboxControl
+          label={__('Open in a new Tab', 'planet4-blocks-backend')}
+          help={__('Open button link in new tab', 'planet4-blocks-backend')}
+          value={attributes.button_link_new_tab}
+          checked={attributes.button_link_new_tab}
+          onChange={toAttribute('button_link_new_tab')}
+        />
+        <TextControl
+          label={__('Articles count', 'planet4-blocks-backend')}
+          help={__('Number of articles', 'planet4-blocks-backend')}
+          type="number"
+          min={1}
+          value={attributes.article_count}
+          onChange={value =>
+            toAttribute('article_count')(Number(value))}
+        />
+        {attributes.posts !== 'undefined' && attributes.posts.length === 0 &&
+          <>
+            <TagSelector
+              value={attributes.tags}
+              onChange={toAttribute('tags')}
+            />
+            <p className="FieldHelp">Associate this block with Actions that have specific Tags</p>
+            <PostTypeSelector
+              label={__('Post Types', 'planet4-blocks-backend')}
+              value={attributes.post_types}
+              onChange={toAttribute('post_types')}
+            />
+            <div className="ignore-categories-wrapper">
+              <CheckboxControl
+                label={__('Ignore categories', 'planet4-blocks-backend')}
+                help={__('Ignore categories when filtering posts to populate the content of this block', 'planet4-blocks-backend')}
+                value={attributes.ignore_categories}
+                checked={attributes.ignore_categories}
+                onChange={toAttribute('ignore_categories')}
               />
             </div>
-          }
-        </PanelBody>
-      </InspectorControls>
-    </Fragment>
-  );
-};
+          </>
+        }
+        {attributes.tags.length === 0 && attributes.post_types.length === 0 &&
+          <div>
+            <hr />
+            <p>{__('Manual override', 'planet4-blocks-backend')}</p>
+            <PostSelector
+              label={__('CAUTION: Adding articles individually will override the automatic functionality of this block. For good user experience, please include at least three articles so that spacing and alignment of the design remains intact.', 'planet4-blocks-backend')}
+              selected={attributes.posts || []}
+              onChange={toAttribute('posts')}
+              postType="post"
+              placeholder={__('Select articles', 'planet4-blocks-backend')}
+              maxLength={10}
+              maxSuggestions={20}
+            />
+          </div>
+        }
+      </PanelBody>
+    </InspectorControls>
+  </>
+);
 
 const renderView = ({attributes, postType, posts, totalPosts}, toAttribute) => {
   const hasMultiplePages = totalPosts > attributes.article_count;
 
   return (
-    <Fragment>
+    <>
```

Comment:
Same as https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1100/files#r1320066948

Reply:
This one has several elements within it so we cannot remove the fragment